saleor/dashboard/customer/views.py

```diff
@@ -28,6 +28,8 @@ def customer_list(request):
     customers, paginator = paginate(customers, 30, request.GET.get('page'))
     ctx = {'customers': customers, 'form': form, 'title': title,
            'paginator': paginator}
+    if not request.GET:
+        ctx.update({'default_pagination_params': {'order_status': 'on'}})
```

Comment:
I mean we should probably fix the view or the form. `request.GET or None` is what causes different behaviour when page number is given. I propose we always pass `request.GET` even when empty and change the default field value to be consistent.